news/2022-09-01-osumonthly-2.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "osu!monthly, August 2022"
+date: 2022-09-01 10:00:00 +0000
+---
+
+Check out a brief overview of what went down in the osu! community for August with the second issue of the osu!monthly!
+
+![](/wiki/shared/news/2022-09-01-osumonthly-2/monthlybanner.jpg)
+
+For those not caught up to speed on what the osu!monthly is, osu!monthly is a reimagining of the hit-sensation osu!weekly, but in a bigger (and dare we say, better) format. Make sure to check out [last month's issue](https://osu.ppy.sh/home/news/2022-08-01-osumonthly-1) if you haven't already!
+
+As this is the second rendition of the recently reawakened project, we are still trying to find a groove and get a sense of what the community likes most and dislikes the least. If you'd like to contribute to any ideas, content, article writing, or anything else that we can feature, please don't be shy and let us know via the [discussion thread on the forums](https://osu.ppy.sh/community/forums/topics/1614314) or get involved in the osu!monthly process over on the [osu!dev Discord server](https://discord.com/invite/ppy).
+
+With all that out of the way, let's get straight into things!
+
+## Table of contents
+
+- [What's new in osu!(lazer)](#what's-new-in-osu!(lazer))
+- [Community recap](#community-recap)
+- [Did you know?](#did-you-know?)
+- [Monthly community quiz](#monthly-community-quiz)
+- [Things to look out for in September](#things-to-look-out-for-in-september)
+- [End word](#end-word)
+
+## What's new in osu!(lazer)
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/RxRwqXmMxd4?list=PLmWVQsxi34bMTmgpDArfxJ32Q0mBLhcE-" frameborder="0" allowfullscreen></iframe>
+
+The weekly updates for osu!(lazer) continue, most notably featuring a change to the osu! slider animations, which should now more closely match the stable version of the game and hopefully minimise the "weird feeling" players have reported when playing the latest version.
+
+Furthermore, a bunch of new features and fixes have been added into the editor, which will make mappers' lives easier as [OliBomby](https://osu.ppy.sh/users/6573093) continues to implement the features from his ever-popular [Mapping Tools](https://osu.ppy.sh/community/forums/topics/940368?n=1) utility directly into osu!(lazer) proper.
```

Comment:
i highly recommend mentioning the slider splitting (coming next release) and merging features, and including a video of them

Reply:
Upon our conversation ([from here](https://discord.com/channels/188630481301012481/218677502141399041/1013863621492752494)) and Blushing's thoughts I decided it's better to leave it for next month (unless by some slim chance it releases before the 1st) and have a portion showing it off within the lazer section of the article.